icons/powerpages/powerpages-original.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 128 128"><defs><linearGradient id="d" x1="17.4" x2="43.08" y1="2.101" y2="94.014" gradientUnits="userSpaceOnUse"><stop stop-color="#A391E8"/><stop offset="1" stop-color="#685CD0"/></linearGradient><linearGradient id="e" x1="94" x2="76.5" y1="86.001" y2="35.501" gradientUnits="userSpaceOnUse"><stop stop-color="#38389F"/><stop offset="1" stop-color="#4B44C0"/></linearGradient><linearGradient id="h" x1="35.21" x2="60.076" y1="-12.482" y2="60.296" gradientUnits="userSpaceOnUse"><stop stop-color="#DBCEF8"/><stop offset="1" stop-color="#C0AFF1"/></linearGradient><linearGradient id="i" x1="21.991" x2="48.766" y1="-3.255" y2="92.369" gradientUnits="userSpaceOnUse"><stop stop-color="#C0AFF1"/><stop offset="1" stop-color="#8675DD"/></linearGradient><clipPath id="a"><path fill="#fff" d="M0 0h96v96H0z"/></clipPath><clipPath id="b"><path fill="#fff" d="M0 0h96v96H0z"/></clipPath><filter id="f" width="1.021" height="1.029" x="-.011" y="-.015"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feBlend in="SourceGraphic" in2="BackgroundImageFix" result="shape"/><feGaussianBlur result="effect1_foregroundBlur" stdDeviation=".4"/></filter><filter id="g" width="1.214" height="1.291" x="-.107" y="-.145"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feBlend in="SourceGraphic" in2="BackgroundImageFix" result="shape"/><feGaussianBlur result="effect1_foregroundBlur" stdDeviation="4"/></filter></defs><g clip-path="url(#a)" transform="matrix(1.3427 0 0 1.34145 -.115 -.1)"><g clip-path="url(#b)"><mask id="c" width="96" height="96" x="0" y="0" maskUnits="userSpaceOnUse"><path fill="#fff" d="M51.002 1.291a5 5 0 0 0-6 0l-42.667 32c-2.666 2-2.666 6 0 8L23.521 57.18l-8.947 6.71c-2.667 2-2.667 6 0 8L45 94.71a5 5 0 0 0 6 0l42.666-32c2.667-2 2.667-6 0-8L72.481 38.821l8.947-6.71c2.666-2 2.666-6 0-8z"/></mask><g mask="url(#c)"><path fill="url(#d)" d="m9.24 67.89 51-38.25L99 58.71 48 96.96Z"/><path fill="url(#e)" d="m23.52 57.18 36.72-27.54 38.759 29.07L65.279 84a5 5 0 0 1-6 0z"/><g filter="url(#f)"><path fill="#000" fill-opacity=".24" d="m-2.998 37.69 51-38.249 38.76 29.07-48 36a5 5 0 0 1-6 0z"/></g><g filter="url(#g)"><path fill="#000" fill-opacity=".32" d="m-2.998 39.29 51-38.249 38.76 29.07-48 36a5 5 0 0 1-6 0z"/></g><path fill="url(#h)" d="m-2.998 37.29 51-38.249 38.76 29.07-48 36a5 5 0 0 1-6 0z"/><path fill="url(#i)" d="M-2.998 37.29 30.722 12a5 5 0 0 1 6 0l35.76 26.82-33.72 25.29a5 5 0 0 1-6 0z"/></g></g></g></svg>
```

Comment:
This icon is missing the shadows that are present in the previous version you had:

| your logo | official logo |
| ----------- | ------------- |
| ![image](https://raw.githubusercontent.com/devicons/devicon/30954dcc2ba89893f8fbea090344406e88cf50fe/icons/powerpages/powerpages-original.svg) | ![image](https://camo.githubusercontent.com/aa7922a52e3df72dec7e421511400f7f72b501765adc8841b68caca6261bf6a5/68747470733a2f2f706f776572617070732e6d6963726f736f66742e636f6d2f696d616765732f6170706c69636174696f6e2d6c6f676f732f7376672f706f77657270616765732e737667) |

Notice the shadows are missing on your version :/

Larger versions so it's easier to see the difference :)
## Your logo
<img src="https://raw.githubusercontent.com/devicons/devicon/30954dcc2ba89893f8fbea090344406e88cf50fe/icons/powerpages/powerpages-original.svg" height="500px" /> 

## Official logo
<img src="https://camo.githubusercontent.com/aa7922a52e3df72dec7e421511400f7f72b501765adc8841b68caca6261bf6a5/68747470733a2f2f706f776572617070732e6d6963726f736f66742e636f6d2f696d616765732f6170706c69636174696f6e2d6c6f676f732f7376672f706f77657270616765732e737667" height="500px" />

![image](https://github.com/devicons/devicon/assets/43886029/97fff46b-78cb-4e79-aed5-a44019b4daae)